CMakeLists.txt

```diff
@@ -146,6 +146,9 @@ elseif(LOCALE MATCHES "^de[_-]")
 elseif(LOCALE MATCHES "^sv[_-]")
   set(STUMPLESS_LANGUAGE "sv-SE")
   set(USE_LOCALE_SV_SE TRUE)
+elseif(LOCALE MATCHES "^sk[_-]")
+  set(STUMPLESS_LANGUAGE "sk-SK")
+  set(USE_LOCALE_SK_ SK TRUE)
```

Comment:
It looks like there is an accidental space between `SK_` and `SK` that will prevent this `set` command from functioning as intended.